ethcore/src/spec/spec.rs

```diff
@@ -67,7 +67,7 @@ pub struct Spec {
 	/// What engine are we using for this?
 	pub engine: Arc<Engine>,
 	/// The fork identifier for this chain. Only needed to distinguish two chains sharing the same genesis.
-	pub fork_name: Option<String>,
+	pub data_dir: String,
```

Comment:
This seems to break compatibility with previous chain specs.
Also the doc string here is now wrong.